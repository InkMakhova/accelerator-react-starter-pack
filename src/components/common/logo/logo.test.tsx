import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import Logo from './logo';

it('should render Logo correctly', () => {
  const {container} = render(
    <Router>
      <Logo />;
    </Router>);
  expect(container).toMatchSnapshot();
});
