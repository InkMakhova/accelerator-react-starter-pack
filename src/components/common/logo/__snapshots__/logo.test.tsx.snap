// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Logo correctly 1`] = `
<div>
  <a
    class="header__logo logo"
    href="/"
  >
    <img
      alt="Логотип"
      class="logo__img"
      height="70"
      src="/img/svg/logo.svg"
      width="70"
    />
  </a>
  ;
</div>
`;
